unstructured/ingest/main.py

```diff
@@ -555,9 +565,20 @@ def main(
                 standard_config=standard_config,
                 config=SimpleS3Config(
                     path=remote_url,
+                    recursive=recursive,
                     access_kwargs={"anon": s3_anonymous},
                 ),
             )
+        elif protocol in ("gs", "gcs"):
+            from unstructured.ingest.connector.gcs import GcsConnector, SimpleGcsConfig
+
+            doc_connector = GcsConnector(  # type: ignore
+                standard_config=standard_config,
+                config=SimpleGcsConfig(
+                    path=remote_url,
+                    recursive=recursive,
```

Comment:
how does this handle cases where we need any authorization to access the gcs bucket? do we need to pass through access_kwargs?

Reply:
I'm not sure how we are handling authorization yet. I'll look into it.

Reply:
Authorization token added. By default it should be none. But it is available now if needed.

```
if token=None (default), GCSFS will attempt to use your default gcloud credentials or, attempt to get credentials from the google metadata service, or fall back to anonymous access. This will work for most users without further action.

Reply:
yea, probably not individuals, but definitely suspect think there will be enterprise users that will need this (and ...we need it for CI)